src/runtime.rs

```diff
@@ -83,16 +83,38 @@ impl TokioTaskAborterFromTrio {
 
 #[pyclass]
 pub(crate) struct FutureIntoCoroutine(
-    Option<Pin<Box<dyn Future<Output = PyResult<()>> + Send + 'static>>>,
+    Option<Pin<Box<dyn Future<Output = PyResult<PyObject>> + Send + 'static>>>,
 );
 
 impl FutureIntoCoroutine {
     pub fn new<F>(fut: F) -> Self
     where
-        F: Future<Output = PyResult<()>> + Send + 'static,
+        F: Future<Output = PyResult<PyObject>> + Send + 'static,
     {
         FutureIntoCoroutine(Some(Box::pin(fut)))
     }
+
+    pub fn from<F, R>(fut: F) -> Self
```

Comment:
Sadly, no because I use the `new` method on specific case.
One example is when I need to convert a `Vec<u8>` into `PyBytes` before converting it to `PyObject`